internal/test/scorecard/common_utils.go

```diff
@@ -177,6 +177,16 @@ func logEvents(r *scapiv1alpha3.TestResult, client *CryostatClientset, namespace
 	return nil
 }
 
+func deferLogWorkloadEvents(r *scapiv1alpha3.TestResult, client *CryostatClientset, namespace string, name string) {
```

Comment:
Maybe rename this to something like `logWorkloadEventsOnError` or something similar. It just reads a bit strangely with `defer defer...`